recipes/cpptrace/all/conanfile.py

```diff
@@ -60,6 +63,8 @@ def generate(self):
         tc = CMakeToolchain(self)
         if is_msvc(self):
             tc.variables["USE_MSVC_RUNTIME_LIBRARY_DLL"] = not is_msvc_static_runtime(self)
+        if Version(self.version) >= Version("0.4.0"):
+            tc.variables["CPPTRACE_USE_EXTERNAL_ZSTD"] = True
```

Comment:
No `self.requires` for zstd?

Reply:
Thank you for noticing that, I'm realizing I should rework something in cpptrace's cmake. Cpptrace doesn't actually need zstd, that's needed by libdwarf.

Reply:
I made a change upstream and applied the patch here